solr/core/src/test/org/apache/solr/handler/TestSampleDocumentsLoader.java

```diff
@@ -63,7 +58,7 @@ public void testJson() throws Exception {
   @Test
   public void testCsv() throws Exception {
     ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set(CSV_MULTI_VALUE_DELIM_PARAM, "\\|");
+    params.set(DefaultSampleDocumentsLoader.CSV_MULTI_VALUE_DELIM_PARAM, "\\|");
```

Comment:
yea this was me just removing the static imports. It makes it clear that this isn't a constant in the `TestSampleDocumentsLoader` class to me. Otherwise it looks like a field in the class.